src/ray/object_manager/object_manager.cc

```diff
@@ -939,9 +939,9 @@ std::string ObjectManager::DebugString() const {
   result << "\n- num buffered profile events: " << profile_events_.size();
   result << "\n- num chunks received total: " << num_chunks_received_total_;
   result << "\n- num chunks received total failed: " << num_chunks_received_total_failed_;
-  result << "\n  - num chunks received cancelled: " << num_chunks_received_cancelled_;
-  result << "\n  - num chunks received thrashed: " << num_chunks_received_thrashed_;
-  result << "\n  - num chunks received, plasma error : "
+  result << "\n- num chunks received cancelled: " << num_chunks_received_cancelled_;
```

Comment:
Could you revert the indentation? It was nice to have because these rows are a subset of the total failed.

Reply:
Oh let me fix this in the other PR.